packages/jellyfish-transaction/src/script/dftx/dftx_governance.ts

```diff
@@ -146,3 +146,27 @@ export class CCreateVoc extends CCreateProposal {
   static OP_CODE = 0x45 // 'E'
   static OP_NAME = 'OP_DEFI_TX_CREATE_VOC'
 }
+
+export type VoteDecision = 0x01 | 0x02 | 0x03
```

Comment:
please treat us nicely
```suggestion
export type VoteDecision = 0x01 | 0x02 | 0x03 // VoteYes | VoteNo | VoteNeutral
```

Reply:
ahah my bad

Reply:
`yay | nay | idk` :P